ROS/osr_control/osr_control/rover.py

```diff
@@ -24,10 +24,10 @@ def __init__(self):
         self.declare_parameters(
             namespace='',
             parameters=[
-                ('rover_dimensions.d1', None),
                 ('rover_dimensions.d2', None),
                 ('rover_dimensions.d3', None),
                 ('rover_dimensions.d4', None),
+                ('rover_dimensions.d1', None),
```

Comment:
Seems like you flipped order here, I think 1,2,3,4 made more sense :)

Reply:
Absolutely! Didn't even notice I did that; fixed in commit 7bbf40e